### PR TITLE
FEATURE: (Un)assign an existing topic from/to an item (see #32, #34)

### PR DESCRIPTION
**Pull request by CorentinLthr-old**
## Content

This includes two features: 
- Assign an existing topic to an item (see #32)
- Unassign a topic from an item (see #34)

*Please note that this contribution does not include the unit tests for these new features.*

---

## Checklist

Please check that your pull request is correct:

- Each commit:
    - [x] corresponds to a contribution that should be notified to users,
    - [x] does not generate new errors or warnings at compile or test time,
    - [x] must be attributed to its real authors (with correct GitHub IDs and [correct syntax for multiple authors](https://help.github.com/articles/creating-a-commit-with-multiple-authors/)).
- The title of a commit should:
    - [x] begin with a contribution type
        - `FEATURE` for a behaviour allowing a user to do something new,
        - `FIX` for a behaviour which has been changed in order to meet user’s expectations,
        - `TEST` when it concerns an acceptance test,
        - `PROCESS` for a change in the way the software is built, tested, deployed,
        - `DOC` when it concerns only internal documentation (however it is better to combine it with the contribution that required this documentation change),
    - [x] be followed by a colon (`:`) with one space after and no space before,
    - [x] be followed by a title as short, as user-centered and as explicit as possible
        - If it is a feature, the title must be the user action (beginning with a verb, and please not `manage`),
        - If it is a fix, the title must describe the intended behavior (with `should`).
    - [x] ends with a reference to the corresponding ticket with the following syntax:
        - `(closes #xx)` if xx is a feature ticket (and the commit is a complete implementation),
        - `(fixes #xx)` if xx is a fix ticket (and the commit is a complete fix),
        - `(see #xx)` otherwise,
- Each committed line is:
    - [x] useful (it would not work if removed)
        - if it is a comment line, its information could not be conveyed by better variables and function naming, better code structuring, or better commit message,
    - [x] related to this very contribution (feature, fix...),
    - [x] in English (with the exception of Gherkin scenarios in French and resulting steps),
    - [x] without any typo in variable, class or function names,
    - [x] correctly indented (spaces rather than tabs, same number of characters as in the rest of the file).

### CODE REVIEW

File: src/components/Item/Item.jsx [hunk 1]
```diff
@@ -74,37 +78,246 @@ class Item extends Component {
...
+      })
+      .catch(error => console.log(`error : ${error}`));
+  }
+
+
+  _removeTopic(topicToDelete) {
+    let params = this.props.match.params;
+    let hypertopic = new Hypertopic(conf.services);
+
+    if (confirm('Voulez-vous vraiment retirer le topic de cette oeuvre ?')) {
```
Comment (benel) [inline src/components/Item/Item.jsx:117]: Should `react-confirm` (or a similar package) be imported?

Comment (benel) [reply src/components/Item/Item.jsx:117]: It doesn't seem to work with `react-confirm`... Or is it just `window.confirm()`?

Comment (florentpergoud) [reply src/components/Item/Item.jsx:117]: You are right, it's window.confirm() which is tried to be used here.

Comment (benel) [reply src/components/Item/Item.jsx:117]: I had to update `react-scripts` and new versions require `window` members to be prefixed. It's OK, I changed the code myself.

**Comment by benel** [comment]
Pull request approved (after a few edits). Now in v7 branch. Congrats!

**Pull request closed without merging**